wandb/sdk/internal/internal_api.py

```diff
@@ -53,6 +55,20 @@
 
 logger = logging.getLogger(__name__)
 
+
+# Context variable for setting API keys for internal and public apis thread-locally
+class _ThreadLocalApiKey(threading.local):
```

Comment:
i wonder if we should call this `_ThreadLocalSettings` - where the only allowed setting right now is api_key?

Reply:
done